pkg/operator/starter.go

```diff
@@ -97,8 +189,45 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 
 	klog.Info("Starting controllerset")
 	go csiControllerSet.Run(ctx, 1)
-
+	go scController.Run(ctx, 1)
 	<-ctx.Done()
 
 	return fmt.Errorf("stopped")
 }
+
+func generateStorageClass(storageDomainName string) *storagev1.StorageClass {
+	reclaimPolicy := corev1.PersistentVolumeReclaimDelete
+	var expected = &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "ovirt-csi-sc",
+			Namespace: defaultNamespace,
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "StorageClass",
+			APIVersion: "storage.k8s.io/v1",
+		},
+		// ObjectMeta will be filled below
+		Provisioner:          instanceName,
+		Parameters:           map[string]string{"storageDomainName": storageDomainName, "thinProvisioning": "true"},
+		ReclaimPolicy:        &reclaimPolicy,
+		MountOptions:         []string{},
+		AllowVolumeExpansion: boolPtr(false),
+	}
+	expected.Annotations = map[string]string{
+		"storageclass.kubernetes.io/is-default-class": "true",
```

Comment:
While this is OK when creating a new storage class during cluster installation, you should make sure you don't overwrite `is-default-class": "false"` (or no annotation at all) when user removes it - they may have set a different default storage class.

Reply:
good point

Reply:
I think it's covered, since I use the object I that I get now